error_handler/error_handler.c

```diff
@@ -3,10 +3,16 @@
 #include "error_handler.h"
 #include "../globals/globals.h"
```

Comment:
```suggestion
#include "globals.h"
````
Hauria de funcionar amb el -I en el compilador